src/Kernel/client/ExecutionPathVisualizer/metadata.ts

```diff
@@ -11,11 +11,8 @@ export interface Metadata {
     x: number;
     /** Array of y coords of control registers. */
     controlsY: number[];
-    /** Array of y coords of target registers. */
-    targetsY: number[];
-    /** y coords of target registers separated by classical registers.
-     *  This is used for rendering multi-qubit unitary gates. */
-    groupedTargetsY?: number[][];
+    /** Array of y coords (or groups of y coords) of target registers. */
+    targetsY: (number | number[])[];
```

Comment:
Nice! Is it worth documenting here when `targetsY` should be `number[]` vs. `number[][]`? I think it should be `number[][]` in the case where `type` is `GateType.ControlledUnitary`, and `number[]` in all other cases? (Partially just asking to check my own understanding... 😊)